### PR TITLE
Upgrade repo tools and regenerate READMEs.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: scripts/generate [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+'use strict';
+
+const childProcess = require('child_process');
+const fs = require(`fs`);
+const path = require(`path`);
+
+const PROJECT_ROOT = path.join(__dirname, `..`);
+
+// Dedupe package.json dependencies
+// WARNING: This will fail if two different versions of the same package are required.
```
Comment (ace-n) [inline scripts/generate]: Nit: is it worth adding a check for this?

File: scripts/generate [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    .filter((dir) => fs.statSync(path.join(directory, dir)).isDirectory())
+    .forEach((dir) => {
+      findSamples(path.join(directory, dir), depth - 1);
+    });
+};
+
+findSamples(PROJECT_ROOT, 3);
+
+// Update root-level package.json (by shelling to npm)
+// const spawn = require('child_process').spawn;
```
Comment (ace-n) [inline scripts/generate]: I assume this is supposed to be commented out?

Comment (jmdobry) [inline scripts/generate]: Will remove

File: scripts/generate (hunk 1 above)
Comment (jmdobry) [inline scripts/generate]: Actually will remove, it's leftover because some of this was copied from your unify script.

**Pull request merged**